### PR TITLE
Release v.1.0.0

### PR DESCRIPTION
**Pull request by OstapMelnychuk**
### Summary
This PR is for uploading the Nextcloud integration app in the Mattermost repository

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine.
```
Comment (hanzei) [inline .circleci/config.yml:1]: Would you please migrate to GitHub actions as the use of CircleCI is depreacted at Mattermost? You can use https://github.com/mattermost/mattermost-app-google-drive/blob/master/.github/workflows/ci.yml as an example.

File: function/install/manifest.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{
+  "app_id": "nextcloud",
+  "version": "v1.0.0",
```
Comment (hanzei) [inline function/install/manifest.json:3]:
```suggestion
  "version": "1.0.0",
```

**Review by hanzei — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (prokhorind) [reply .circleci/config.yml:1]: sure 
https://github.com/OstapMelnychuk/mattermost-app-nextcloud-release/pull/1

Comment (OstapMelnychuk) [reply .circleci/config.yml]: Fixed

File: function/install/manifest.json (hunk 2 above)
Comment (OstapMelnychuk) [reply function/install/manifest.json]: Fixed

**Review by hanzei — APPROVED** [review]

**Pull request merged**